public/imgs/brt_black_yellow.svg

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
The icons that are on the map are different than the ones in the selector on the dashboard on the bottom - is this intentional? If they refer to the same thing they should be the same resource.